firebase-database/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 # 20.1.0
 * [unchanged] Updated to accommodate the release of the updated
 [database] Kotlin extensions library.
+* [fixed] Remove redundant gradle sourceSet to clear spurious Android Studio unbound reference errors.
```

Comment:
Not sure if we need a changelog here, as this isn't a user-facing change.

Reply:
Yes, no need of changelog entries for non-user visible changes.